cucumber-expressions/java/pom.xml

```diff
@@ -36,7 +36,7 @@
         <dependency>
             <groupId>junit</groupId>
             <artifactId>junit</artifactId>
-            <version>4.12</version>
+            <version>4.13-beta-3</version>
```

Comment:
We could use JUnit Jupiter instead. I think that is preferable to using a beta version.

Reply:
Eh JUnit Jupiter - is that a new API I have to learn?

Reply:
I don't think there is much to risk using a beta version for a test dependency

Reply:
> Eh JUnit Jupiter - is that a new API I have to learn?

Also known as JUnit 5. For the sort of tests we are using, you'd have to use some new imports, that is it.

I'll update it to Junit 5 on the next pass through. It's not so difficult.